### PR TITLE
Add circleci config

### PR DESCRIPTION
**Pull request by niil-ohlin**
We is migrating our ci services to circle-ci, starting with our open source projects. This PR adds a circle-ci config file.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+version: 2
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+jobs:
+  test-iOS:
+    macos:
+      xcode: "9.3.0"
```
Comment (saidsikira) [inline .circleci/config.yml:9]: Can we run it with Xcode 10?

**Review by saidsikira — COMMENTED** [review]
Nice! ~Would it be possible to run this PR on CircleCI?~

**Comment by saidsikira** [comment]
Ahh nice I see that it ran successfully on CircleCI!

File: .circleci/config.yml (hunk 1 above)
Comment (niil-ohlin) [reply .circleci/config.yml:9]: Sure. I did not want to change anything in the config right now. I can do it in another PR.

**Review by egarni — APPROVED** [review]

**Pull request merged**